docs/Design/Rest Parameters.md

```diff
@@ -0,0 +1,143 @@
+# Rest Parameters
+
+A parameter that creates an array for all unassigned arguments.
+
+
+## Commands
+
+### `rest parameters`
+
+`rest parameters : type name`
+
+Creates a rest parameter.
+Name of the array created for the unassigned arguments.

```

Comment:
When I said "'name is the name of the...' is redundant text", I was a little unclear. It would have been more reasonable to say that the "The" is a little redundant, and having two instances of the word "name" can be a little confusing.

The new form ("Name of the array created for the unassigned arguments.") doesn't actually make any sense. If I give confusing feedback please do question it.